pylint/checkers/match_statements_checker.py

```diff
@@ -46,6 +63,19 @@ class MatchStatementChecker(BaseChecker):
             "Emitted when there is more than one sub-pattern for a specific "
             "attribute in a class pattern.",
         ),
+        "R1905": (
+            "Use '%s' instead",
+            "match-class-bind-self",
+            "Match class patterns are faster if the name binding happens "
+            "for the whole pattern and any lookup for `__match_args__` "
+            "can be avoided.",
+        ),
+        "R1906": (
+            "Use keyword attributes instead of positional ones",
```

Comment:
```suggestion
            "Use keyword attributes instead of positional ones ('%s')",
```
I think we can construct the expected here too. (And one day use it to autofix?)